tests/terraform/checks/resource/gcp/example_CloudMySqlLocalInfileOff/main.tf

```diff
@@ -0,0 +1,123 @@
+
+                resource "google_sql_database_instance" "fail" {
```

Comment:
@JamesWoolfenden could align the TF code here, it looks super strange 😄

Reply:
done